app/forms/provider_interface/offer_wizard.rb

```diff
@@ -2,19 +2,26 @@ module ProviderInterface
   class OfferWizard
     include ActiveModel::Model
 
-    STEPS = { make_offer: %i[select_option conditions check] }.freeze
+    STEPS = { make_offer: %i[select_option conditions check],
+              change_offer: %i[select_option providers courses study_modes locations conditions check] }.freeze
```

Comment:
Isn't locations supposed to be the step before `study_modes`? 🤔

Reply:
No. The study mode is required in order to limit the course options. The alternative would be the candidates picking a course based on where it's located and then potentially that course not having a full time option available. We discussed this with both Michael and Adam as we also thought it would be the other way around.

Actually as part of this we also need to reverse the order of Study mode and Location in all places where offers are displayed, e.g. make offer summary page, offer page etc, but we decided to do that separately.